api/v1alpha1/compression_types.go

```diff
@@ -41,6 +41,17 @@ type Compression struct {
 	// +required
 	Type CompressorType `json:"type"`
 
+	// ChooseFirst indicates whether to choose the first encoding type
+	//
+	// +optional
+	ChooseFirst bool `json:"chooseFirst,omitempty"`
```

Comment:
Could you expand on the description here so it's easier for users to understand?